### PR TITLE
rubocop update

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -68,7 +72,12 @@ Style/SingleLineBlockParams:
 Style/TrivialAccessors:
   Enabled: false
 
+
```
Comment (lslezak) [inline .rubocop.yml]: NP: looks like an extra empty line...

File: src/ruby/yast/builtins.rb [hunk 2]
```diff
@@ -350,7 +346,7 @@ def self.flatten(value)
 
       value.reduce([]) do |acc, i|
         return nil if i.nil?
-        acc.push(*Yast.deep_copy(i))
+        acc.concat(Yast.deep_copy(i))
```
Comment (lslezak) [inline src/ruby/yast/builtins.rb]: Oh, nice catch! :+1:

Comment (jreidinger) [inline src/ruby/yast/builtins.rb]: it is actually rubocop auto fix :)

**Comment by lslezak** [comment]
LGTM

**Pull request merged**